Source/WebKit/Shared/ApplePay/WebPaymentCoordinatorProxy.messages.in

```diff
@@ -24,6 +24,7 @@
 
 #if ENABLE(APPLE_PAY)
 
+// Currently DispatchedTo=Networking when APPLE_PAY_REMOTE_UI and DispatchedTo=UI when !APPLE_PAY_REMOTE_UI
 [
     DispatchedFrom=WebContent,
```

Comment:
Is it not possible to express something like this? Would be handy to bake this into the specification rather than through a comment!

[
    DispatchedFrom=WebContent,
#if ENABLE(APPLE_PAY_REMOTE_UI)
    DispatchedTo=Networking,
#else
    DispatchedTo=UI,
#endif
    EnabledBy=ApplePayEnabled
]